src/sql/READ_ONLY/CPM_ACTION_LOG_TABLE.sql

```diff
@@ -0,0 +1,25 @@
+-- SQL to create log table for CPM script actions.
+
+-- CTools
+
+CREATE TABLE CTOOLS_USER.CPM_ACTION_LOG (
+       ACTION_TIME DATE,
```

Comment:
DATE datatype is defaulted to 19-Dec-16 in the database. Time stamp is not included. It would be good to have the timestamp along with date